server/txSender/dataFormatter.go

```diff
@@ -68,8 +69,19 @@ func (df *dataFormatter) createRegisterBridgeOperationsData(bridgeData *sovereig
 	registerBridgeOpData := []byte(registerBridgeOpsPrefix +
 		"@" + hex.EncodeToString(bridgeData.AggregatedSignature) +
 		"@" + hex.EncodeToString(bridgeData.Hash))
+	registerBridgeOpData = append(registerBridgeOpData, hashesHexEncodedArgs...)
 
-	return append(registerBridgeOpData, hashesHexEncodedArgs...)
+	bridgeDataArgs := []byte(
+		"@" + hex.EncodeToString(bridgeData.PubKeysBitmap) +
```

Comment:
It's mandatory to have bitmap + epoch before `hashesHexEncodedArgs` so the contract can decode the arguments properly.

Reply:
Fixed